### PR TITLE
Remove import React and Update CartPage, HomePage and set title, favicon website

### PR DESCRIPTION
**Pull request by NAPATKRUP**
Remove import React and Update CartPage(Table with grid), HomePage(Add Logo in Banner) and set title, favicon website

### CODE REVIEW

File: frontend/public/manifest.json [hunk 1]
```diff
@@ -1,23 +1,6 @@
 {
-  "short_name": "React App",
-  "name": "Create React App Sample",
-  "icons": [
-    {
-      "src": "favicon.ico",
-      "sizes": "64x64 32x32 24x24 16x16",
-      "type": "image/x-icon"
-    },
```
Comment (Icyscools) [inline frontend/public/manifest.json]: You should add icons for PWAs supported.

File: frontend/src/components/home/components/BannerWrapper.tsx [hunk 2]
```diff
@@ -2,8 +2,15 @@ import { FC } from 'react';
...
-      Home Page Banner
+    <div className="h-80 grid lg:grid-cols-2 bg-dark-100 rounded-2xl">
+      <div className="flex justify-center items-center">
+        <img src="/logo.png" className="w-1/2" alt="logo" />
+      </div>
+      <div className="flex justify-center items-center">
+        <p className="lg:text-2xl md:text-lg text-white-100">
+          อยากเพิ่มเสน่ห์ในตัวคุณต้องนึกถึงเรา
+        </p>
+      </div>
```
Comment (Icyscools) [inline frontend/src/components/home/components/BannerWrapper.tsx:13]: Is it dummy?

Comment (NAPATKRUP) [reply frontend/src/components/home/components/BannerWrapper.tsx:13]: it for real lol

File: frontend/public/manifest.json (hunk 1 above)
Comment (NAPATKRUP) [reply frontend/public/manifest.json]: okay

File: frontend/src/components/home/components/BannerWrapper.tsx (hunk 2 above)
Comment (Icyscools) [reply frontend/src/components/home/components/BannerWrapper.tsx:13]: ok bro.

**Comment by Icyscools** [comment]
Nice job @NAPATKRUP!

**Pull request merged**